README.md

```diff
@@ -1 +1,217 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!--video description-->
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> It will deploy soon.
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+Open terminal on the same folder of the project and run:
+
+```sh
+ npm install
+```
+
+Initialize npm install webpack loaclly, and install the webpack-cli
+
+```sh
+ npm install webpack webpack-cli --save-dev
+```
+
+setting up HtmlWebpackPlugin
+
+```sh
+ npm install --save-dev html-webpack-plugin
+```
+
+To build and run the project from /src to /dist use
+
+```sh
+ npm run build
+```
+
+To run the project automatically loading up the page use
+
+```sh
+ npm start
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/brhanuhailu/Leaderboard.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd leaderboard
+  npm install
+```
+
+### Usage
+
+To run the project, open index.html in /dist directory with browser
+
+### Run tests
+
+To run tests, run the following command:
+
+To check for html errors run:
+
+```sh
+  npx hint .
+```
+
+To check for css errors run:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+To check for js errors run:
+
+```sh
+  npx eslint .
+```
+
+### Deployment
+
+It will be Deployed using Github Pages soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Brhanu Hailu**
+
+- GitHub: [@brhanuhailu](https://github.com/brhanuhailu)
+- Twitter: [@tigraycountry](https://twitter.com/TigrayCountry)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/brhanu-hailu-85578a246/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Add function for Clear all list.
+
+- [ ] put,Get, Post and cut data from API
+- [ ] Use ES6 Modularity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/brhanuhailu/Leaderboard/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> I am so excited to accept you feedback to improve my profession.
+
+If you like this project like it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+we would like to thank Microverse organization
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
```

Comment:
- [ ] Kindly add a license to your README as it defines the terms by which your app might be used and it gives your project more professionalism.